apis/r/DESCRIPTION

```diff
@@ -34,7 +34,7 @@ Imports:
     Matrix,
     stats,
     bit64,
-    tiledb (>= 0.21.2),
+    tiledb (>= 0.21.3),
```

Comment:
We can also consider leaving it as is. By pointing at the very installation sources we point at, version 0.21.3 will be installed once available (as it is the highest version).  By putting constraints up like this we effectively only risk breaking builds where the constraint is not yet satisfied, i.e. at CRAN it can take days to get macOS builds, and we also have seen Conda builds delayed or failing for various reasons.  

Luckily we currently only depend on r-universe which is quicker.  But I am wary of putting up constraints when we don't really have to.